### PR TITLE
Add first draft of readme

### PR DESCRIPTION
**Pull request by davisshaver**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -8,107 +7,44 @@ Stable tag: 4.3
...
-if the `/trunk/readme.txt` file says that the stable tag is `4.3`, then it is `/tags/4.3/readme.txt` that'll be used
-for displaying information about the plugin.  In this situation, the only thing considered from the trunk `readme.txt`
-is the stable tag pointer.  Thus, if you develop in trunk, you can update the trunk `readme.txt` to reflect changes in
-your in-development version, without having that information incorrectly disclosed about the current stable version
-that lacks those changes -- as long as the trunk's `readme.txt` points to the correct stable tag.
-
-    If no stable tag is provided, it is assumed that trunk is stable, but you should specify "trunk" if that's where
-you put the stable version, in order to eliminate any doubt.
+== Description==
+Used alongside [Shortcake](https://wordpress.org/plugins/shortcode-ui/), Shortcake Bakery adds a fine selection of shortcodes to your WordPress site.
```
Comment (danielbachhuber) [inline readme.txt]: Is it worthwhile to note that the shortcodes work without Shortcake?

File: readme.txt [hunk 2]
```diff
@@ -8,107 +7,44 @@ Stable tag: 4.3
...
-Here's a link to [WordPress](http://wordpress.org/ "Your favorite software") and one to [Markdown's Syntax Documentation][markdown syntax].
-Titles are optional, naturally.
-
-[markdown syntax]: http://daringfireball.net/projects/markdown/syntax
-            "Markdown is what the parser uses to process much of the readme file"
+== Changelog ==
 
-Markdown uses email style notation for blockquotes and I've been told:
-> Asterisks for *emphasis*. Double it up  for **strong**.
+= 0.1.0 (July 16, 2015) =
```
Comment (danielbachhuber) [inline readme.txt]: I typically leave the date as `???` until the actual release date, just as a FYI

File: readme.txt [hunk 3]
```diff
@@ -8,107 +7,44 @@ Stable tag: 4.3
...
-= 0.5 =
-This version fixes a security related bug.  Upgrade immediately.
-
-== Arbitrary section ==
+1. Shortcodes are accessible through the Insert Element screen, exposed in the media library.
 
-You may provide arbitrary sections, in the same format as the ones above.  This may be of use for extremely complicated
-plugins where more information needs to be conveyed that doesn't fit into the categories of "description" or
-"installation."  Arbitrary sections will be shown below the built-in sections outlined above.
+2. Shortcodes preview in the visual editor, with a quick edit button to provide easy access to attribute fields.
```
Comment (danielbachhuber) [inline readme.txt]: Might be helpful to include screenshots of each shortcode.

File: readme.txt (hunk 1 above)
Comment (davisshaver) [inline readme.txt]: @danielbachhuber Good point, that had slipped my mind writing this.

**Comment by davisshaver** [comment]
@danielbachhuber How do you think this looks? Last issue I see is making the images work correctly, not sure what's going on there.

File: readme.txt [hunk 4]
```diff
@@ -8,107 +7,63 @@ Stable tag: 4.3
...
-This section describes how to install the plugin and get it working.
-
-e.g.
-
-1. Upload `plugin-name.php` to the `/wp-content/plugins/` directory
-1. Activate the plugin through the 'Plugins' menu in WordPress
-1. Place `<?php do_action('plugin_name_hook'); ?>` in your templates
+It's a plugin! Install it like any other. 
+
+The follow shortcodes are now available for your use within the content field:
```
Comment (danielbachhuber) [inline readme.txt]: Should we mention these shortcodes in the Description, instead of installation?

File: readme.txt [hunk 5]
```diff
@@ -8,107 +7,63 @@ Stable tag: 4.3
...
+
+- Image Comparison `[image-comparison left="9" right="10" position="center"]`
+- Facebook `[facebook url="https://www.facebook.com/willpd/posts/1001217146572688"]`
+- iFrames (requires some configuration) `[iframe src="http://www.buzzfeed.com"]`
+- Infogram `[infogram url="http://infogr.am/washington_marijuana_sales"]`
+- PDF's `[pdf url="http://www.gpo.gov/fdsys/pkg/BILLS-114hr2048enr/pdf/BILLS-114hr2048enr.pdf"]`
```
Comment (danielbachhuber) [inline readme.txt]: Hm, this example doesn't seem to work.

**Comment by danielbachhuber** [comment]
> How do you think this looks?

Looks pretty good.

> Last issue I see is making the images work correctly, not sure what's going on there.

They'll start working once we do the sync to WordPress.org

File: readme.txt (hunk 5 above)
Comment (davisshaver) [inline readme.txt]: @danielbachhuber Yeah, I'm not sure what PDF to use. Needs to allow cross origin access. I don't think it's a big deal if the actual PDF file doesn't work.

**Comment by davisshaver** [comment]
I added note about the Access-Control-Allow-Origin and put in my copy of the SCOTUS pdf. I guess an arbitrary, working URL is better than an arbitrary, non-working URL?

File: readme.txt [hunk 6]
```diff
@@ -8,107 +7,63 @@ Stable tag: 4.3
...
-your in-development version, without having that information incorrectly disclosed about the current stable version
-that lacks those changes -- as long as the trunk's `readme.txt` points to the correct stable tag.
-
-    If no stable tag is provided, it is assumed that trunk is stable, but you should specify "trunk" if that's where
-you put the stable version, in order to eliminate any doubt.
+- Image Comparison `[image-comparison left="9" right="10" position="center"]`
+- Facebook `[facebook url="https://www.facebook.com/willpd/posts/1001217146572688"]`
+- iFrames (requires some configuration) `[iframe src="http://www.buzzfeed.com"]`
+- Infogram `[infogram url="http://infogr.am/washington_marijuana_sales"]`
+- PDF's (requires PDF with `Access-Control-Allow-Origin` header) `[pdf url="https://assets.fusion.net/edit/pdfs/the_interview_budget_excerpts.pdf"]`
```
Comment (danielbachhuber) [inline readme.txt]: "requires PDF served from domain with..." to be more clear

File: readme.txt [hunk 7]
```diff
@@ -8,107 +7,63 @@ Stable tag: 4.3
...
+- Playbuzz `[playbuzz url="https://www.playbuzz.com/Fusion/5-mind-blowing-facts-about-cloning-from-jurassic-park-youll-never-believe-actually-exist-in-real"]`
+- Rap Genius `[rap-genius]`
+- Scribd `[scribd url="http://www.scribd.com/doc/269993358/Transgender-Care-Memorandum"]`
+- Scripts (requires some configuration) `[script src="https://ajax.googleapis.com/ajax/libs/threejs/r69/three.min.js"]`
```
Comment (danielbachhuber) [inline readme.txt]: "requires some configuration" might be more clear as "requires code-level configuration of accepted domains"

Should we have a call for contributions?

File: readme.txt [hunk 8]
```diff
@@ -1,6 +1,5 @@
 === Shortcake Bakery ===
 Contributors: fusionengineering
```
Comment (danielbachhuber) [inline readme.txt]: `fusionengineering, davisshaver, danielbachhuber`

File: shortcake-bakery.php [hunk 9]
```diff
@@ -2,10 +2,10 @@
 /*
 Plugin Name: Shortcake Bakery
 Version: 0.1-alpha
-Description: PLUGIN DESCRIPTION HERE
-Author: YOUR NAME HERE
-Author URI: YOUR SITE HERE
-Plugin URI: PLUGIN SITE HERE
+Description: A fine selection of Shortcake-powered shortcodes.
+Author: fusionengineering
```
Comment (danielbachhuber) [inline shortcake-bakery.php]: `fusionengineering, davisshaver, danielbachhuber`

**Pull request merged**